db/migrate/20150316135303_create_course_enrol_requests.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCourseEnrolRequests < ActiveRecord::Migration
+  def change
+    create_table :course_enrol_requests do |t|
```

Comment:
can we just call it course_registrations or something? enrollment is the list of people in something. you register/apply to enroll into something.

@jsyeo thoughts?